src/components/MonsterDetails.js

```diff
@@ -1,27 +1,33 @@
 import React from 'react'
-import styled from 'styled-components'
+import MonsterDetailsHead from './MonsterDetailsHead'
 import MonsterDetailsStat from './MonsterDetailsStats'
+import Proficiencies from './Proficiencies'
+import ActionsAbilities from './ActionsAbilities'
 
 export default function MonsterDetails({ monster }) {
   return (
     <>
-      <Headline>{monster.name}</Headline>
+      <MonsterDetailsHead
+        name={monster.name}
+        type={monster.type}
+        cr={monster.challenge_rating}
+        size={monster.size}
+        alignment={monster.alignment}
+      />
       <MonsterDetailsStat
         strength={monster.strength}
         dexterity={monster.dexterity}
         constitution={monster.constitution}
         intelligence={monster.intelligence}
         wisdom={monster.wisdom}
         charisma={monster.charisma}
-      ></MonsterDetailsStat>
+      />
+      <Proficiencies proficiencies={monster.proficiencies} />
```

Comment:
due to your naming `MonsterDetailsHead` and `MonsterDetailsStat` : 
```suggestion
      <MonsterProficiencies proficiencies={monster.proficiencies} />
```